src/i18n/ru/ui.ts

```diff
@@ -17,4 +17,55 @@ export default UIDictionary({
 	'articleNav.prevPage': 'Назад',
 	'leftSidebar.learnTab': 'Изучить',
 	'leftSidebar.referenceTab': 'Ссылки',
+	'404.content': 'Эта страница не в нашей солнечной системе.',
+	'404.linkText': 'Доставь меня домой.',
+	'404.title': 'Не найдено',
+	'a11y.sectionLink': 'Раздел, озаглавленный',
+	'a11y.skipLink': 'Перейти к контенту',
+	'checklist.or': 'или',
+	'cms.navTitle': 'Больше гайдов по CMS',
+	'contributors.seeAll': 'Посмотреть всех контрибьюторов',
+	'deploy.altSectionTitle': 'Больше гайдов по деплою',
+	'deploy.filterLabel': 'Отфильтровать по типу деплоя',
+	'deploy.sectionTitle': 'Гайды по деплою',
+	'deploy.ssrTag': 'SSR',
+	'deploy.staticTag': 'Static',
+	'fallbackContent.linkText': 'Узнать больше о том, как вы можете помочь',
+	'fallbackContent.notice':
+		'Эта страница ещё не доступна на вашем языке, поэтому вы видите английскую версию. Вы можете помочь перевести её!',
+	'feedback.button': 'Оставьте отзыв',
+	'fileTree.directoryLabel': 'Директория',
+	'footer.privacyPolicy': 'Политика конфиденциальности',
+	'install.autoTab': 'Автоматический CLI',
+	'install.manualTab': 'Ручная настройка',
+	'integrations.adapters': 'SSR адаптеры',
+	'integrations.changelog': 'Список изменений',
+	'integrations.footerTitle': 'Больше интеграций',
+	'integrations.others': 'Другое',
+	'integrations.renderers': 'UI фреймворки',
+	'leftSidebar.a11yTitle': 'Главное',
+	'leftSidebar.sponsoredBy': 'Спонсоры',
+	'leftSidebar.viewInEnglish': 'Посмотреть на английском языке',
+	'migration.navTitle': 'Больше инструкций по миграции',
+	'multipleChoice.defaultCorrect': 'Верно!',
+	'multipleChoice.defaultIncorrect': 'Попробуйте ещё раз!',
+	'multipleChoice.submitLabel': 'Подтвердить',
+	'navbar.a11yTitle': 'Верх',
+	'progress.done': 'Готово',
+	'progress.todo': 'Что нужно сделать',
+	'rightSidebar.a11yTitle': 'Вторичное',
+	'rightSidebar.github': 'Документация Astro на Github',
+	'rightSidebar.overview': 'Краткий обзор',
```

Comment:
I’m seeing this translated as “quick”, “short” or “brief” overview, but in English we just say “Overview” without an adjective. In reality, this heading just links to the top of the page so we might want to choose something more neutral than quick/brief.

Would just “обзор” without the adjective make sense?

```suggestion
	'rightSidebar.overview': 'обзор',
```

Reply:
Well, kind of. In russian "overview" actually means "short overview". We don't have the exact word for "overview". "Обзор" means something more abstract and common, but it's ok.